src/appleseed/renderer/modeling/camera/camera.cpp

```diff
@@ -295,16 +295,6 @@ Vector2d Camera::extract_film_dimensions() const
     return film_dimensions;
 }
 
-namespace
-{
-    // Compute the focal length (in meters), given the film width (in meters)
-    // and the horizontal field of view (in degrees).
```

Comment:
The comment has already been moved to the header file. For comments like for code, we try to avoid duplication so method descriptions should only appear in the public ("interface") part of the code, usually in the header files.